src/components/footer.js

```diff
@@ -1,19 +1,78 @@
 import React from "react"
 import PropTypes from "prop-types"
+import Section from "../styles/Section"
 import styled from "styled-components"
 import Logo from "./logo"
 import Img from "gatsby-image"
 
+const GrayBackground = styled.div`
+  background: #eaecf0;
+`
+
 const SizedLogo = styled(Logo)`
```

Comment:
`It looks like you've wrapped styled() around your React component (SquareLogo), but the className prop is not being passed down to a child. No styles will be rendered unless className is composed within your React component.` is the error being thrown by React here. I have a `size` prop on the Logo component that will let you set the size, so you can create the component by doing something like `size="40px"` to get the proper size. This logo is the square logo though so you can only set 1 side of it.

Reply:
Sounds good! I'll refactor this once yours is merged!